packages/react-docsite-components/src/components/Page/Page.types.ts

```diff
@@ -112,6 +112,22 @@ export interface IPageProps<TPlatforms extends string = string> {
 
   /** Currently selected platform. */
   platform?: TPlatforms;
+
+  /**
+   * The available versions whose documentation is presented on the website.
+   */
+  versionOptions?: IContextualMenuItem[];
+
+  /**
+   * Callback that determines what happens when a version is chosen from the version dropdown.
+   */
+  onVersionMenuClick?: (event: React.MouseEvent<HTMLElement, MouseEvent>, item: IContextualMenuItem) => void;
+
+  /**
+   * The data regarding the current version of the library.
```

Comment:
Might be good to make clear that this is a package.json object. Or even better, pull out and pass through only the needed info ahead of time.